src/pybind/mgr/influx/module.py

```diff
@@ -205,6 +226,14 @@ def send_to_influx(self):
                               self.config['username'])
                 client.create_database(self.config['database'])
             else:
+                self.set_health_checks({
+                    'MGR_INFLUX_SEND_FAILED': {
+                        'severity': 'warning',
+                        'summary': 'Failed to send data to Influx due to a '
```

Comment:
s/a Exception/an exception/

Maybe be a bit more specific and say "an InfluxDB client exception"

Reply:
Actually after looking at 20198 as well, I think it's better for these messages not to mention exceptions and just to say "Failed to send data to InfluxDB" or similar -- the important information is that we failed to send something, the exception is just the mechanism by which we found out.